dev.py

```diff
@@ -722,6 +722,40 @@ class Test(Task):
         'task_dep': ['build'],
     }
 
+    @staticmethod
+    def run_lcov(dirs):
```

Comment:
I'll note that the question on removing old coverage data is also answered by the `runtests.py` code that this came from: https://github.com/scipy/scipy/blob/maintenance/1.9.x/runtests.py#L510-L518